app/api/v1/endpoints/strategies.py

```
@@ -1200,4 +1200,100 @@ async def update_strategy_submission(
+        logger.exception("Failed to toggle strategy", user_id=str(current_user.id))
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail="Failed to toggle strategy"
+        ) from e
+
+
+@router.put("/{strategy_id}/config")
+async def update_strategy_configuration(
+    strategy_id: str,
+    config: Dict[str, Any],
+    current_user: User = Depends(get_current_user)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)